.github/workflows/pytorch110_aarch64.yml

```diff
@@ -70,4 +69,4 @@ jobs:
         set -e
         echo "Try to deploy runtime docker image..."
         source $HOME/.cache/proxy_config
-        bash ./scripts/ci/deploy_tf_wrapper.sh
\ No newline at end of file
+        bash ./scripts/ci/deploy_tf_wrapper.sh
```

Comment:
Nit: can we rename the script name `deploy_tf_wrapper.sh` as well?

Reply:
OK, rename `deploy_tf_wrapper.sh` into `deploy_wrapper.sh`